appimage-build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 APP="syncthing-gtk"
+APPID="me.kozec.syncthingtk"
```

Comment:
That's `syncthingtk` as in `syncthing-tk` or `syntchin-gtk`? Perhaps this is a clever contraction, but I'd suggested `net.syncthing.syncthing-gtk`, or `github.kozec.syncthing-gtk` or some such.

Reply:
Me too, but this app already uses that as its D-Bus name; they should be the same.